### PR TITLE
Use common logging provider

### PR DESCRIPTION
**Pull request by pgrawehr**
Followup of #1332: Use the new common logging provider for all bindings that used logging

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/1497)

### CODE REVIEW

File: src/devices/Arduino/ArduinoBoard.cs [hunk 1]
```diff
@@ -67,12 +72,14 @@ public ArduinoBoard(string portName, int baudRate)
         {
             _dataStream = null;
             _serialPort = new SerialPort(portName, baudRate);
+            _logger = this.GetCurrentClassLogger();
         }
 
         /// <summary>
-        /// Attach to this event to retrieve log messages. The Exception argument may be null if it is only an informational message.
+        /// The board logger.
+        /// TODO: Move to base class
```
Comment (krwq) [inline src/devices/Arduino/ArduinoBoard.cs:80]: no TODOs in the comments please

File: src/devices/Common/Iot/Device/Common/DebuggerOutputLogger.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
+﻿using System;
```
Comment (krwq) [inline src/devices/Common/Iot/Device/Common/DebuggerOutputLogger.cs:1]: copyright headers missing in couple of files

**Review by Ellerbach — APPROVED** [review]
Thanks for all the changes!

**Review by krwq — APPROVED** [review]

**Pull request merged**